src/ralph/utils.py

```diff
@@ -84,3 +84,16 @@ def now():
     """Return the current UTC time in ISO format"""
 
     return datetime.datetime.now(tz=datetime.timezone.utc).isoformat()
+
+
+def nested_get(dictionary, keys):
+    """Get the nested dict value by keys array."""
+
+    if keys is None:
```

Comment:
```suggestion
    if keys is None or not len(keys):
```

Reply:
Good catch! I haven't foreseen this case. Thanks

Reply:
This part has been undone as by using `reduce`, it's returning `None` for empty lists.